examples/plot_pole_ladder_s2.py

```diff
@@ -62,7 +62,7 @@ def main():
 
     base_point = gs.to_ndarray(base_point, to_ndim=2)
     origin = gs.concatenate(
-        [base_point, base_point, final_geodesic(gs.array([0]))], axis=1)
+        [base_point, base_point, final_geodesic(gs.array([0.]))], axis=1)
```

Comment:
Hopefully, we can get rid of these soon, using `convert_to_wider_dtype` backend functions.

Reply:
The problem here is more that path except a 1d array for t, a single scalar cannot be passed.

Reply:
I added a to_ndarray to deal with this case

Reply:
Yes, but also `gs.array([0])` -> `gs.array([0.])`, right?